.github/workflows/downstream_tests.yml

```diff
@@ -0,0 +1,46 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+    branches: [main]
```

Comment:
Maybe instead of every PR we can make it both as workflow dispatch (i.e. manual trigger) and dependency of publish to pypi?
I am not sure if you intend to add scikit-lego as well, but test there typically take between 2 and 5 mins

Reply:
yeah that might be better

Reply:
> I am not sure if you intend to add scikit-lego as well

I will take it as a _yes_ 😂
![image](https://github.com/user-attachments/assets/c15d8cae-7887-46f1-9400-caf8b9d98875)

Reply:
altair actually takes less than our own tests 😄 I think we might as well keep that on in CI

and for scikit-lego, perhaps some tests these could be marked "slow", and we just skip those here?

Reply:
Agree! I think it is enough to run everything without extra dependencies as in [our scheduled ci](https://github.com/koaning/scikit-lego/blob/main/.github/workflows/schedule-dependencies.yml#L30C12-L30C104)

TL;DR

```
run: pytest -n auto --disable-warnings --cov=sklego -m "not cvxpy and not formulaic and not umap"
```